src/core/edit/edit.cell.view.d.ts

```diff
@@ -0,0 +1,60 @@
+import {Shortcut} from '../infrastructure/shortcut';
+import {CellEditor} from './edit.cell.editor';
+import {getFactory as valueFactory} from '../services/value';
+import {getFactory as labelFactory} from '../services/label';
+import {Model} from "../infrastructure/model";
+import {Table} from "../dom/table";
+import {CommandManager} from "../infrastructure/command.manager";
+import {ICell} from "../cell/cell";
+import {INoopResult} from "../utility/utility";
+import {IEditorOptions} from "../column-type/column.model";
+
+export declare interface IContextFactory{
+	column: ICell.column,
+	row: ICell.row,
+	columnIndex: ICell.columnIndex,
+	rowIndex: ICell.rowIndex,
+	oldValue: ICell.value,
+	newValue: ICell.value,
+	oldLabel: ICell.label,
+	newLabel: ICell.label,
+	unit: string,
+	tag: any,
+	valueFactory: valueFactory,
+	labelFactory: labelFactory
+}
+
+export declare class EditCellView {
+	constructor(public model: Model, public table: Table, public commandManager: CommandManager);
+
+	editor: CellEditor;
+	shortcut: Shortcut;
+	commands: Map;
+	shortcutOff: boolean;
```

Comment:
```shortcutOff``` is a function which removes shortcuts registered before.
It has interface ```(): boolean```
Besides, you should add ```register``` function to shortcut.d.ts. And this function returns interface which I mentioned above.